sdk/ai/azopenaiassistants/README.md

```diff
@@ -0,0 +1,114 @@
+# Azure OpenAI assistants client module for Go
+
+NOTE: this client can be used with Azure OpenAI and OpenAI.
+
+Azure OpenAI Service provides access to OpenAI's powerful language models including the GPT-4, GPT-35-Turbo, and Embeddings model series, as well as image generation using DALL-E.
```

Comment:
Does this need to be reworded? It reads like it's from `azopenai`.

Reply:
Yeah, the readme is still a work in progress....